packages/flutter/lib/src/rendering/shifted_box.dart

```diff
@@ -628,71 +635,70 @@ class RenderConstrainedOverflowBox extends RenderAligningShiftedBox {
   }
 }
 
-/// Renders a box, imposing no constraints on its child, allowing the child to
-/// render at its "natural" size.
+/// A [RenderBox] that applies an arbitrary transform to its [constraints]
```

Comment:
Might be worthwhile to document somewhere in here what this does if child is null.